packages/@blockprotocol/type-system-web/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@local/tsconfig/node.json",
+  "extends": "@local/tsconfig/dom.json",
```

Comment:
Fixes missing `WebAssembly` in packages/@blockprotocol/type-system-{node,web}dist/index.d.ts`